src/python/install.sh

```diff
@@ -444,18 +443,19 @@ install_from_source() {
     find_version_from_git_tags VERSION "https://github.com/python/cpython"
 
     # Some platforms/os versions need modern versions of openssl installed
-    # via common package repositories, for now rhel-7 family, use case statement to 
+    # via common package repositories, for now rhel-7 family, use case statement to
     # make it easy to expand
+    INSTALL_PATH="/usr"
```

Comment:
```suggestion
    SSL_INSTALL_PATH="/usr/local"
```

Can we rename this variable? As `INSTALL_PATH` is used in many different places, want to avoid accidental overrides.